datasets/nell/README.md

```diff
@@ -0,0 +1,234 @@
+* candidate_source: A free-form amalgamation of more specific provenance information describing the justification(s) NELL has for possibly believing this category or relation instance.
+
+For the nell_belief_sentences and nell_candidate_sentences, we have extracted the underlying sentences, sentence count and URLs and provided a shortened version of the entity, relation and value field by removing the string "concept:" and "candidate:". There are two types of sentences, 'CPL' and 'OE', which are generated by two of the modules of NELL, pattern matching and open web searching, respectively. There may be duplicates. The configuration is as follows:
+* entity: The Entity part of the (Entity, Relation, Value) tripple. Note that this will be the name of a concept and is not the literal string of characters seen by NELL from some text source, nor does it indicate the category membership of that concept
+* relation: The Relation part of the (Entity, Relation, Value) tripple. In the case of a category instance, this will be "generalizations". In the case of a relation instance, this will be the name of the relation.
+* value: The Value part of the (Entity, Relation, Value) tripple. In the case of a category instance, this will be the name of the category. In the case of a relation instance, this will be another concept (like Entity).
+* probability: A confidence score for the belief. Note that NELL's scores are not actually probabilistic at this time.
+* sentence: the raw sentence. For 'CPL' type sentences, there are "[[" "]]" arounds the entity and value. For 'OE' type sentences, there are no "[[" and "]]".
+* url: the url if there is one from which this sentence was extracted
+* count: the count for this sentence
+* sentence_type: either 'CPL' or 'OE'
+
+### Data Splits
+
+There are no splits. 
+
+## Dataset Creation
+
+### Curation Rationale
+
+This dataset was gathered and created over many years of running the NELL system on web data.
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+See the research paper on NELL. NELL searches a subset of the web
+(Clueweb09) and the open web using various open information extraction
+algorithms, including pattern matching.
+
+#### Who are the source language producers?
+
+The NELL authors at Carnegie Mellon Univiersty and data from Cluebweb09 and the open web. 
+
+### Annotations
+
+#### Annotation process
+
+The various open information extraction modules of NELL.
+
+#### Who are the annotators?
+
+Machine annotated.
+
+### Personal and Sensitive Information
+
+Unkown, but likely there are names of famous individuals.
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+The goal for the work is to help machines learn to read and understand the web.
+
+### Discussion of Biases
+
+Since the data is gathered from the web, there is likely to be biased text and relationships.

```

Comment:
```suggestion
Since the data is gathered from the web, there is likely to be biased text and relationships.

[More Information Needed]
```
this section will probably require further details

You can just add `[More Information Needed]`

Reply:
Done